appng-appngizer/src/main/java/org/appng/appngizer/controller/SiteController.java

```diff
@@ -152,11 +156,18 @@ private boolean supportsReloadFile(org.appng.api.model.Site site) {
 	}
 
 	@DeleteMapping(value = "/site/{name}")
-	public ResponseEntity<Void> deleteSite(@PathVariable("name") String name) throws BusinessException {
+	public ResponseEntity<Void> deleteSite(@PathVariable("name") String name, ServletContext context)
```

Comment:
There already is a protected `ServletContext` in the base class `ControllerBase`, so you don't need it as a method parameter.